test/internal/testenv/testenv.go

```diff
@@ -46,6 +46,15 @@ func KongEffectiveVersion() string {
 	return os.Getenv("TEST_KONG_EFFECTIVE_VERSION")
 }
 
+// KongHelmChartVersion is the 'kong' helm chart version to use in tests.
+func KongHelmChartVersion() string {
+	v := os.Getenv("TEST_KONG_HELM_CHART_VERSION")
+	if v == "" {
+		os.Exit(1)
+	}
```

Comment:
Probably would be useful to give some more feedback in case it occurs, WDYT? Maybe simply `fmt.Println("ERROR: missing required TEST_KONG_HELM_CHART_VERSION")`?